.bazelrc

```diff
@@ -0,0 +1 @@
+build --action_env=BAZEL_CXXOPTS="-std=c++17"
```

Comment:
Let's also add a `.bazelversion` file, similarly to e.g. https://github.com/tensorflow/tensorflow/blob/master/.bazelversion.